pkg/burner/burner.go

```diff
@@ -89,7 +89,19 @@ func prepareRootfs(s *schema.SystemSpec, fs vfs.FS, tempOverlayfs string) error
 				return err
 			}
 		}
+
+		if s.EnsureCommonDirs {
+			info(":steaming_bowl: Ensure commons dirs.")
+			ensureDirs(tempOverlayfs)
+		}
+
 	} else if len(s.Packages.Rootfs) > 0 {
+
+		if s.EnsureCommonDirs {
```

Comment:
Disjoin this `if` and do this operation just once

Reply:
sorry, but after two months it's a bit hard to follow the speech. Is not executed one time the method prepareRootfs?
What means `do this operation just once`?

Reply:
Maybe the speech is: do you want to permit to define the list of the packages to install also with image? If yes we could move  EnsureCommonsDir later and only one time.